{"name": "test_login_user_with_issues[locked_out_user-secret_sauce-Sorry, this user has been locked out.]", "status": "passed", "parameters": [{"name": "user", "value": "'locked_out_user'"}, {"name": "password", "value": "'secret_sauce'"}, {"name": "errormsg", "value": "'Sorry, this user has been locked out.'"}], "start": 1686518887178, "stop": 1686518889979, "uuid": "2ae8a0d6-a11c-4f36-94d5-8cd93c3ca5c1", "historyId": "7ccd7299ae53bcf97946e1bf6ac467be", "testCaseId": "b7b2b8804ad6016478cd3229692aa3b8", "fullName": "test.test_login.TestSignIn#test_login_user_with_issues", "labels": [{"name": "tag", "value": "p1"}, {"name": "tag", "value": "@pytest.mark.usefixtures('driver_setup')"}, {"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestSignIn"}, {"name": "host", "value": "M-X14WXMDVVX-9.local"}, {"name": "thread", "value": "77846-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_login"}]}